design/open/spark-outputcommitter/lakeFS-output-committer-execution-plan.md

```diff
@@ -0,0 +1,38 @@
+# LakeFS Output Committer - Execution Plan
+[LakeFS Output Committer Proposal](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md)
+
+## Overview
+Implementing lakeFS Output Committer meaning implementing the class [FileOutputCommitter](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.html) and using lakeFS client to create/merge branches as part of setup and commit job/tasks (this is the [sample flow](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md#sample-flow))
+
+### Milestone 1
+The first milestone is focused on writing a file with single mode (probably text) and one file format (probably append).
+At this stage we'll implement `commit`, `setup`, `abort` class methods.
```

Comment:
What's the definition of done and the time estimate (in person's weeks) of each milestone? 
At which point are we planning to add integration tests?

Reply:
I think "testing" at the bottom of M1 will have this.  Whether this is component or unit or system or integration is TBD, and IMO should be decided as part of "testing".

Reply:
"testing"  is ok with me, and the bottom of M1 means after M1 or as part of it?